libraries/stdlib/stdlib_defs.mtlx

```diff
@@ -1100,6 +1100,17 @@
     <output name="out" type="vector4" default="0.0, 0.0, 0.0, 0.0" />
   </nodedef>
 
+  <!--
+    Node: <bump>
+    Offset the surface normal by a scalar value.
+  -->
+  <nodedef name="ND_bump_vector3" node="bump" nodegroup="houdini">
```

Comment:
Maybe nodegroup "math" or "convolution2d" here?

Reply:
@jstone-lucasfilm  this was actually meant to be "geometric", since it sort of lives with the other geometry-related nodes; but I can change it to math or convolution2d if those would be better.